src/app/collection/config/collection-config.component.html

```diff
@@ -18,83 +18,88 @@
       </icon-picker>
     </div>
 
-    <input class="color-class name edit-border"
-           [(ngModel)]="collection.name"
-           (blur)="updateCollection()"
-           [style.color]="collection.color"
-           i18n-title="@@header.collection.name"
-           title="Edit name of the file">
+    <div class="color-class name flex-grow-2 mr-4 cursor-text"
+         [textContent]="collection.name"
+         (blur)="name.scrollTo(0,0); onNewColectionName(name.textContent)"
+         [style.color]="collection.color"
+         contenteditable="true"
+         placeholder="Collection name"
+         [attr.spellcheck]="false"
+         i18n-title="@@header.collection.name"
```

Comment:
There should be `i18n` attribute for `placeholder` as well.

Reply:
Yes..thanks